includes/forms/class-llms-forms.php

```diff
@@ -130,10 +132,23 @@ public function are_usernames_enabled() {
 	 */
 	private function block_to_field_settings( $block ) {
 
+		$is_visible = $this->is_block_visible( $block );
+		/**
+		 * Filters whether or not invisible fields should be included
+		 *
+		 * If the block is not visible (according to LLMS block-level visibility settings)
+		 * it will return an empty array (signaling the field to be removed).
+		 *
+		 * @param $filter Whether or not invisible fields should be included. Default is `false`.
+		 */
+		if ( ! $is_visible && apply_filters( 'llms_filter_out_invisible_field', false ) ) {
```

Comment:
"filter_out" feels programmy and "hide" seems to be more readibly understandable... I think?

Also, prefixing the filter with "forms" like many of the other hooks in this area of the codebase seems to make it less prone to future collisions.

```suggestion
		if ( ! $is_visible && apply_filters( 'llms_forms_hide_invisible_field', false ) ) {
```

Reply:
we decided to name it `'llms_forms_remove_invisible_field'`

Reply:
I missed it

Reply:
Fixed review :arrow_up: